src/instrumentation/opentelemetry.ts

```diff
@@ -15,12 +15,18 @@
  */
 
 import { Version } from "@/version.js";
-import opentelemetry, { SpanStatusCode, TimeInput } from "@opentelemetry/api";
+import { api } from "@opentelemetry/sdk-node";
 import { FrameworkSpan, GeneratedResponse } from "./types.js";
+import { BaseAgent } from "@/agents/base.js";
+import { Tool } from "@/tools/base.js";
+import { BaseLLM } from "@/llms/base.js";
+import os from "os";
 
-export const tracer = opentelemetry.trace.getTracer("bee-agent-framework", Version);
+const name = "bee-agent-framework";
 
-export const activeTracesMap = new Map<string, string>();
+export const meter = api.metrics.getMeter(name, Version);
```

Comment:
I'd assume name and version are taken from the OTEL SDK instance. Is that not the case?